### ISSUE TITLE
Build fails with AGP 8.0.2

### ISSUE THREAD

**Issue by dvoloshyn** [QuisApp/flutter_contacts#123]
When upgrading from AGP 7.4.2 to AGP 8.0.2 build fails with the following error:
```
* What went wrong:
A problem occurred configuring project ':flutter_contacts'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Please specify a namespace in the module's build.gradle file like so:

     android {
         namespace 'com.example.namespace'
     }

     If the package attribute is specified in the source AndroidManifest.xml, it can be migrated automatically to the namespace value in the build.gradle file using the AGP Upgrade Assistant; please refer to https://developer.android.com/studio/build/agp-upgrade-assistant for more information.
```
Should be fairly simple to fix, right?

**Comment by felixkriechbaum** [comment]
Can you attach an
`flutter doctor -v`

and your build gradle?

**Comment by dvoloshyn** [comment]
Hi Felix,

I believe that someone is already working on fixing this issue in PR https://github.com/QuisApp/flutter_contacts/pull/121
So, the question mainly is when this work can be done, merged and released

**Comment by joachim-quis** [comment]
Should be fixed in version [1.1.7](https://pub.dev/packages/flutter_contacts/versions/1.1.7)
